wpeverest/user-registration
# UR-499 Global settings input validation

### All Submissions:






### Changes proposed in this Pull Request:

This PR introduces a validation layer for all global setting input submissions.
Also, few tooltip changes are made.

Jira Ticket: [UR-499](https://themegrill.atlassian.net/browse/UR-499)

### How to test the changes in this Pull Request:

1. Try to save global settings with invalid inputs.
2. Verify if invalid values are being saved and whether error is thrown or not.
3. Also, verify if tooltip changes are reflected.

### Types of changes:




### Other information:




### Changelog entry

> UR-499 Global settings input validation


[UR-499]: https://themegrill.atlassian.net/browse/UR-499?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
